bin/node-template/runtime/src/lib.rs

```diff
@@ -1015,6 +1025,162 @@ impl darwinia_tron_backing::Trait for Runtime {
 
 impl darwinia_header_mmr::Trait for Runtime {}
 
+// dvm needed
+/// Fixed gas price of `1`.
+pub struct FixedGasPrice;
+
+impl FeeCalculator for FixedGasPrice {
+	fn min_gas_price() -> U256 {
+		// Gas price is always one token per gas.
+		0.into()
+	}
+}
+
+parameter_types! {
+	pub const ChainId: u64 = 43;
+}
+
+pub struct ConcatAddressMapping;
+
+impl AddressMapping<AccountId> for ConcatAddressMapping {
+	fn into_account_id(address: H160) -> AccountId {
+		let suffix = &BlakeTwo256::hash(&address[..])[0..12];
+
+		let mut data = [0u8; 32];
+		data[0..20].copy_from_slice(&address[..]);
```

Comment:
@wuminzhe please document detail dvm mapping address specification, considering adding specific prefix to indentify dvm mapping address from normal subkey address.

Reply:
Maybe a checksum to prevent collision?

Reply:
Done.